emails/migrations/0051_abusemetrics_num_email_forwarded_per_day.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.14 on 2022-08-29 20:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("emails", "0050_profile_store_phone_log"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="abusemetrics",
+            name="num_email_forwarded_per_day",
+            field=models.PositiveSmallIntegerField(default=0),
```

Comment:
You'll need to allow `null` or add a database-level default to avoid errors when you go to production. There are CircleCI tests for this, but they were not present in `2022.08.16.1`.